src/pages/Login.js

```diff
@@ -1,7 +1,131 @@
-import React from 'react'
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+import loginApi from "../utils/loginApi";
+
+const Container = styled.section`
+  display: flex;
+  justify-content: center;
+`;
+
+const Content = styled.div`
+  display: flex;
+  margin-top: 128px;
+  height: 450px;
+  width: 600px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: rgb(245, 245, 245);
+`;
+
+const Form = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 75%;
+`;
+
+const Title = styled.h1`
+  font-size: 2em;
+`;
+
+const Hr = styled.hr`
+  margin: 16px 0;
+  width: 80%;
+  height: 1px;
+  background-color: rgb(211, 211, 212);
+  border: none;
+`;
+
+const Input = styled.input`
+  font-size: 1rem;
+  margin-bottom: 16px;
+  margin-top: 8px;
+  border: 1px solid rgba(204, 204, 204, 0.8);
+  line-height: 1.5;
+  padding: 8px;
+`;
+
+const Button = styled.button`
+  margin-top: 16px;
+  width: 100%;
+  background-color: rgb(32, 52, 73);
+  color: white;
+  border: 0px;
+  height: 50px;
+  cursor: pointer;
+  transition: all 0.3s ease 0s;
+`;
+
+const SignInButton = styled.button`
+  padding: 8px;
+  display: flex;
+  font-size: 16px;
+  margin-bottom: 32px;
+`;
+
+const ImageContainer = styled.div`
+  width: 16px;
+`;
+
+const Image = styled.img`
+  max-width: 100%;
+`;
 
 export const Login = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const history = useHistory();
+
+  const handleLogin = async () => {
```

Comment:
the idea of this Login module for this particular case is to "protect" the whole page, so if the user is not logged in, it should see this screen